config/locales/en.yml

```diff
@@ -618,6 +630,21 @@ en:
       applications: *applications
       applications_tooltip: Number of open applications
 
+  payments:
+    create:
+      page_title: &page_title '%{payment_type} Payment by: %{user_name}'
+      title: *page_title
+      something_wrong: Something went wrong - please contact system administrator
+
+    update:
+      status_uncertain: Payment status uncertain - please contact system administrator
```

Comment:
Good thought, and I added a "TODO" comment.  

This brings up the question as to whether we should let all (or some) flash messages persist, rather than go away after a few seconds.  So, for instance, adding a mailto link in these messages would be almost useless currently, since the user barely has time to read the message (if they see it all), much less click on an embedded link.

Perhaps we just need to add easy access to a mailto link to the SHF admin(s), present on every page (maybe in a footer).

FYI - the second message (status_uncertain) is not used any more so I removed it.